src/Ting/Serializer/DateTimeImmutable.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/***********************************************************************
+ *
+ * Ting - PHP Datamapper
+ * ==========================================
+ *
+ * Copyright (C) 2014 CCM Benchmark Group. (http://www.ccmbenchmark.com)
+ *
+ ***********************************************************************
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ **********************************************************************/
+
+namespace CCMBenchmark\Ting\Serializer;
+
+class DateTimeImmutable implements SerializerInterface
+{
+    /**
+     * @var array
+     * format => Always used for serialization. Used for unserialization only if unSerializeUseFormat is true
+     * unSerializeUseFormat => if false, any valid datetime format is automatically used
+     * @see http://php.net/manual/en/datetime.formats.compound.php
+     */
+    private static $defaultOptions = ['format' => 'Y-m-d H:i:s', 'unSerializeUseFormat' => true];
```

Comment:
I used format `c` for serialized but could not use it for unserialization since this format in unknown to method `\DateTimeImmutable::createFromFormat()`